native/src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -121,58 +134,46 @@ export default (
         const { message, key, allAvailableLanguages, newsId, type, ...rest } = action.params
         return {
           ...state,
-          newsRouteMapping: {
-            ...state.newsRouteMapping,
+          routeMapping: {
+            ...state.routeMapping,
             [key]: allAvailableLanguages
-              ? { status: 'languageNotAvailable', type, allAvailableLanguages, ...rest }
-              : { status: 'error', message, newsId, type, ...rest }
+              ? { routeType: 'news', status: 'languageNotAvailable', type, allAvailableLanguages, ...rest }
+              : { routeType: 'news', status: 'error', message, newsId, type, ...rest }
           }
         }
       }
+      case 'CLEAR_CATEGORY':
+      case 'CLEAR_EVENT':
       case 'CLEAR_NEWS': {
```

Comment:
there is no `CLEAR_POI`?

Reply:
Could you add it or create an issue? Maybe we could also use a generic action for this.

Reply:
Year something like CLEAR_ROUTE could make sense.